nest/02.introduction.md

```diff
@@ -0,0 +1,57 @@
+Now, if you want that application to work properly, you'll also need to install the package they used to encrypt messages.
+
+If a package is required by another one to work properly, that package is called a **dependency**.
+Whenever you install a package via a package manager (and Nest as well), it will check if the package in question has any dependencies.
+And if it does, it will install only the dependencies that are not already installed on the OS.
+So, when you install a package, don't be surprised if you see other packages getting installed too, they are just dependencies needed for that package to work properly.
+
+## 1.3 Repositories
+Packages are stored in repositories.
+Think of a repository as a big folder containing the packages.
+Within each repository, there are a certain number of categories, and a package has to belong to one, and only one of them.
+The categories are a way to distinguish packages with the same name, but having different purposes.
+This means that we can have two packages with the same name, in the same repository but not in the same category, coexisting together, without any issue.
+
+[//]: # (TODO: Add link to the section 3.1)
+For now, all you need to know is that there are official and unofficial repositories for Nest.
+Only `stable`, which is an official repository, is enabled by default.
+This repository contains all the packages that are considered stable and safe to use.
+The repositories will be more detailed in [a future section]().
+
+## 1.4 Package name syntax
+[//]: # (TODO: add link to the section 3.3)
+The packages names are represented in the following way by Nest: `repository::category/package#[version|version_req]` (*version_req* means *version requirement*).
+The concept of version requirement will be detailed in [this section]().
+If you don't specify any version or version requirement, the latest one available will be installed by default.
+
+However, most parts of this syntax may be omitted if it matches **one** and only **one** package.
+In a way, the only real mandatory part of this syntax is the package name itself. That means you can use:
+* `repository::package`, if that package is the only one with its name in the whole repository.
+* `category/package`, if the package's name is unique in this specific category, and if that category belongs to a single repository used by Nest.
+* `package`, if that package is the only one with its name amongst all the repositories used by Nest.
+
+If you're not sure on the repository, category, or name of a package, you can check on [Raven-OS official website](https://packages.raven-os.org).
+It lists all the packages in the official repositories for Nest.
+Meaning you can see the repository and category they belong to, the latest version released for Nest, and a short description of what they are for.
```

Comment:
`what they are for`? Is that English? What does it mean? :o

Reply:
"à quoi ils servent" in french. Maybe replace that with "what they provide"?
I don't like how this sentence starts with a "meaning". 
I'd replace this and the previous sentence with something like this:
"It lists all the packages in the official repositories for Nest as well as the repository and category they belong to, the latest version released, and a short description of what they provide."